examples/src/main/scala/shapeless/examples/csv.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2014 Mario Pastorelli (pastorelli.mario@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package shapeless.examples
+
+import shapeless._, syntax.singleton._
+
+import scala.util.{Try,Success,Failure}
+
+
+/*
+ * This example shows how to createa a serializer/deserializer from CSV to
+ * products.
+ * */
+
+// The class to serialize or deserialize
+case class Person(name: String, surname: String, age: Int)
```

Comment:
You are right, sorry but I submitted the code yesterday late. I changed the example, now  we deserialize into a list of `Person` by using `Generic`. What do you think?

In future I will try to make possible to create a `CSVConverter[List[Person]]` instead of `CSVConverter[List[personGen.Repr]]` + `Generic[Person]`...it shouldn't be difficult.